packages/kbn-i18n/src/core/known_locales.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const knownLocales = {
```

Comment:
Since we don't use this for i18n purposes anywhere, but mostly for the locale selection for number formatting right now, I wonder if we really should/want to put this into the `i18n` package. If you feel it's the right place, it would be nice if someone from @elastic/kibana-localization could give a quick approval on that.

Reply:
@timroes It makes sense to me having this part inside the `i18n` package. Ideally the whole support for `Intl` including `Intl.NumberFormat` for numbers and currencies would be exposed through `@kbn/i18n` as well.

Reply:
Given this, I will move the currencies list to i18n as well.